UPLOAD/inc/languages/english/purgesoftdeleted.lang.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$l['purgesoftdeleted_task_log'] = "Purge soft deleted posts and threads now are pernamenly deleted";
```

Comment:
Is this necessary given that it's already in the admin language file?

Reply:
Fixed, thx!